db/products.js

```diff
@@ -58,9 +58,11 @@ const updateProduct = async (fields = {}) => {
 
     const setString = Object.keys(fields).map((key, index) => {
         if (key === "imageURL" || key === "inStock") {
-            
+            return `"${key}"=$${index + 1}`;
+        } else {
+            return `${key}=$${index + 1}`
```

Comment:
REQUIRED - Are we adding the value to setString regardless of if it's `imageURL` or `inStock` or not?  Seems like these two if/else cases do almost exactly the same thing?

Reply:
imageURL and inStock have " " around them per psql syntax. I was trying to make sure they had the quotes so there were no bugs. Is this not needed? @wallacepreston